src/utils.jl

```diff
@@ -49,18 +50,18 @@ the region you are going to compare.
        It is intended to be run on a fine wavelength grid, then downsampled to the observational (or 
        otherwise desired) grid.
 """
-function constant_R_LSF(flux::AbstractVector{F}, wls, R) where F <: Real
+function constant_R_LSF(flux::AbstractVector{F}, wls, R; window_size=3) where F <: Real
     #ideas - require wls to be a range object? Use erf to account for grid edges?
     convF = zeros(F, length(flux))
     normalization_factor = Vector{F}(undef, length(flux))
     lb, ub = 1,1 #initialize window bounds
     for i in 1:length(wls)
         λ0 = wls[i]
         σ = λ0 / R / 2
```

Comment:
Shouldn't this be lamdbda/R/2.35 (or 2sqrt(2ln(2)))

Reply:
As usual, @segasai is correct. 

I’m embarrassed I missed this, particularly since — on the same day I reviewed this — I implemented the LSF convolution and resampling with a sparse matrix in Python, and there I used the correct 1/2.35 scaling. 🙃